common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -827,11 +849,7 @@ def get_transcripts_info(self, is_bumper=False, include_val_transcripts=False):
         # For phase 2, removing `include_val_transcripts` will make edx-val
         # taking over the control for transcripts.
         if include_val_transcripts:
-            transcript_languages = get_available_transcript_languages(
-                edx_video_id=self.edx_video_id,
-                youtube_id_1_0=self.youtube_id_1_0,
-                html5_sources=self.html5_sources
-            )
+            transcript_languages = get_available_transcript_languages(edx_video_id=self.edx_video_id)
```

Comment:
@Qubad786 Do we need to take care of TODO below now ?

Reply:
@mushtaqak no, because S3 is still on fallback.